frontend/src/e2e-test/specs/5_employee/reservation-calendar.spec.ts

```diff
@@ -498,6 +498,24 @@ describe('Unit group calendar', () => {
       attendances.length
     )
   })
+
+  test('Employee can add absence records', async () => {
+    const childReservations = (await loadUnitAttendancesSection())
+      .childReservations
+    await calendarPage.selectMode('week')
+    reservationModal = await childReservations.openReservationModal(
+      child1Fixture.id
+    )
+    await reservationModal.addAbsence(mockedToday)
+    await waitUntilEqual(
+      () =>
+        childReservations
+          .childRows(child1Fixture.id)
+          .findAllByDataQa('absence')
```

Comment:
`data-qa` attributes are such a low-level detail that in my opinion they should only be used in the page model and not directly in tests.